pkg/pdp/service/roots_add.go

```diff
@@ -429,10 +430,12 @@ func (p *PDPService) AddRoots(ctx context.Context, id uint64, request []types.Ro
 	// Use clientDataSetId from FilecoinWarmStorageService (not PDPVerifier's setId)
 	// Use nextPieceId as firstAdded (this is what PDPVerifier will pass to the callback)
 	signature, err := p.signingService.SignAddPieces(ctx,
+		p.id,
 		datasetInfo.ClientDataSetId, // Use FilecoinWarmStorageService clientDataSetId
 		nextPieceId,                 // firstAdded is the next piece ID
 		pieceDataBytes,
 		metadata,
+		[][]receipt.AnyReceipt{}, // TODO: extract blob/accept receipts, include pdp/accept receipts
```

Comment:
given all the work to add the acceptance store, this particular TODO seems like a weird one to leave open.